components/integrations/edit_outgoing_webhook/edit_outgoing_webhook.test.tsx

```diff
@@ -2,17 +2,21 @@
 // See LICENSE.txt for license information.
 
 import React from 'react';
-import {shallow} from 'enzyme';
+import {shallow, ShallowWrapper} from 'enzyme';
+
+import {OutgoingWebhook} from 'mattermost-redux/types/integrations';
+
+import EditOutgoingWebhook
+    from 'components/integrations/edit_outgoing_webhook/edit_outgoing_webhook';
 
 import {browserHistory} from 'utils/browser_history';
-import EditOutgoingWebhook from 'components/integrations/edit_outgoing_webhook/edit_outgoing_webhook.jsx';
+import {TestHelper} from '../../../utils/test_helper';
```

Comment:
Nit: Shouldn't need a relative path here, `'utils/test_helper'` should work just fine.

Reply:
fixed it